Tasks/ExtractFilesV1/extractfilestask.ts

```diff
@@ -20,18 +21,33 @@ tl.debug('win: ' + win);
 var xpTarLocation: string;
 var xpUnzipLocation: string;
 // 7zip
-var xpSevenZipLocation: string;
-var winSevenZipLocation: string = path.join(__dirname, '7zip/7z.exe');
+let xpSevenZipLocation: string;
+let defaultWinSevenZipLocation: string = path.join(__dirname, '7zip/7z.exe');
+let winSevenZipLocation: string;
 
 function getSevenZipLocation(): string {
+    if (customPathToSevenZipTool) {
+        tl.debug('Get 7z tool from user defined location');
+        xpSevenZipLocation = customPathToSevenZipTool;
+        winSevenZipLocation = customPathToSevenZipTool;
```

Comment:
Could it make sense just to return the  `customPathToSevenZipTool` here?

Reply:
Yeah it make sense, fixed. Thank you!